pkg/restic/backupper.go

```diff
@@ -170,6 +182,26 @@ ForEachVolume:
 	return volumeSnapshots, errs
 }
 
+func volumeExists(pod *corev1api.Pod, volumeName string) bool {
```

Comment:
Is it worth optimizing this into a single function so we don't have to range over pod.Spec.Volumes twice?

Reply:
```go
func resticCanBackup(log logrus.FieldLogger, pod *corev1api.Pod, volumeName string) bool {
  for _, volume := range pod.Spec.Volumes {
    if volume.Name == volumeName {
      if volume.HostPath != nil {
        // log that it's hostPath
        return false
      }
      return true
    }
  }
  // log that we couldn't find it
  return false
}

Reply:
If it's easier to unit test them separately, it's fine to leave as-is. We know how to optimize and can do it later if needed.